get-influxdb-data.sh

```diff
@@ -217,12 +222,12 @@ else
 fi
 
 typeset QUERY_STRING='SELECT '"${QUERY_VAR_STRING}"' from "'"${INFLUXDB_SERIES}"'" where '"${INFLUXDB_QUERY_WHERE}"' GROUP BY '"${INFLUXDB_QUERY_GROUP}${QUERY_FILL_STRING}" || _fatal "_expandquery failed"
-_verbose curl -G --basic --user "${INFLUXDB_USER}" \
+_verbose winpty curl -G --basic --user "${INFLUXDB_USER}" -o "$OUTPUTFILE" \
 	"https://${INFLUXDB_SERVER}/influxdb:8086/query?${INFLUXDB_OPTPRETTY}" \
 	--data-urlencode "db=${INFLUXDB_DB}" \
-	--data-urlencode "q=$QUERY_STRING"
+	--data-urlencode "q=$QUERY_STRING" 
 
-curl -G --basic --user "${INFLUXDB_USER}" \
+winpty curl -G --basic --user "${INFLUXDB_USER}" -o "$OUTPUTFILE" \
```

Comment:
I forgot that I hacked this for Lex. This absolutely cannot use `winpty` unconditionally, beause this will break on macOS and Linux., Instead, we have to do a variable substition here, based on the system we're running on. The detection has to happen in the front matter. If someone else wants to do this, look at the MCCI scripts (like the release scripts or tools/bin/mcci-autobuild, or mellivora-regress) and see how I usually handle portabilty. I can do this for this script, but it will be some days before I can look at it.